client/spring-zeebe/src/test/java/io/camunda/zeebe/spring/client/bean/MethodInfoTest.java

```diff
@@ -0,0 +1,83 @@
+package io.camunda.zeebe.spring.client.bean;
+
+import io.camunda.zeebe.spring.client.annotation.ZeebeVariable;
+import io.camunda.zeebe.spring.client.annotation.ZeebeWorker;
+import org.junit.Test;
+import org.mockito.internal.util.collections.Sets;
+
+import java.beans.Introspector;
+import java.lang.reflect.Method;
+import java.lang.reflect.Parameter;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MethodInfoTest {
+
+  public static class WithZeebeWorker {
+
+    @ZeebeWorker(type = "bar", timeout = 100L, name = "kermit")
+    public void handle() {
+    }
+  }
+
+  public static class WithZeebeWorkerParamValues {
+
+    @ZeebeWorker(type = "foo", name = "kermit")
+    public void handle(@ZeebeVariable("foo") String foo) {
```

Comment:
Is it necessary to explicitly define the variable name or could we just use the argument name?

Reply:
@falko thank you so fast replay!

Yes, you can't get the real variable name when the code is compiled